substrate/frame/merkle-mountain-range/src/lib.rs

```diff
@@ -59,20 +59,17 @@
 extern crate alloc;
 
 use alloc::vec::Vec;
-use frame_support::weights::Weight;
-use frame_system::pallet_prelude::{BlockNumberFor, HeaderFor};
 use log;
-use sp_mmr_primitives::utils;
-use sp_runtime::{
-	traits::{self, One, Saturating},
-	SaturatedConversion,
-};
 
-pub use pallet::*;
+pub use frame::prelude::*;
```

Comment:
```suggestion
use frame::prelude::*;
```

My point was that we don't expose the `prelude` publicly, because that wasn't exposed before.

Reply:
https://github.com/paritytech/polkadot-sdk/pull/7081/commits/90c7ea17c343aa4150faab69f04d1101a395b292